app/javascript/packs/pw-strength.js

```diff
@@ -96,11 +101,7 @@ function analyzePw() {
   const pwFeedback = document.getElementById('pw-strength-feedback');
   const forbiddenPasswordsElement = document.querySelector('[data-forbidden]');
   const forbiddenPasswords = getForbiddenPasswords(forbiddenPasswordsElement);
-
-  // the pw strength module is hidden by default ("display-none" CSS class)
-  // (so that javascript disabled browsers won't see it)
-  // thus, first step is unhiding it
-  pwCntnr.className = '';
+  const minPasswordLength = +pwCntnr.getAttribute(['data-pw-min-length']);
```

Comment:
I'm surprised this works, since `getAttribute` should expect a string, and this is passing an array.

```suggestion
  const minPasswordLength = +pwCntnr.getAttribute('data-pw-min-length');
```

In testing, I do see that it _does_ work. I think it's probably because it's being internally cast to a string, which happens to have the same outcome.

```js
['data-pw-min-length'].toString()
// 'data-pw-min-length'
```

This reminds me: Can we enhance one of the existing feature specs you'd edited below to include a case where we expect the minimum password error message? In addition to the JavaScript spec for this.